.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cio-native-sdks/customerio-ios"]
+	path = cio-native-sdks/customerio-ios
+	url = https://github.com/customerio/customerio-ios
```

Comment:
We should ideally add branch in here, will help with testing changes on a specific branch.